packages/node/src/integrations/tracing/fastify.ts

```diff
@@ -31,8 +31,10 @@ export const fastifyIntegration = defineIntegration(_fastifyIntegration);
 
 // We inline the types we care about here
 interface Fastify {
-  register: (plugin: unknown) => void;
-  addHook: (hook: string, handler: (request: unknown, reply: unknown, error: Error) => void) => void;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  register: (plugin: any) => void;
```

Comment:
Should we make these `unknown` instead? That forces us to check the things, which I guess is the safer approach? 🤔

Reply:
If we use `unknown` the types don't line up, that's why we switched to any

see: 
![image](https://github.com/getsentry/sentry-javascript/assets/18689448/8aa5ba5a-2bb4-4900-a74d-bdbdcf0c6508)